lib/rmt/cli/smt_importer.rb

```diff
@@ -1,11 +1,18 @@
 require 'csv'
 require 'ostruct'
+require 'fast_gettext'
+require 'initializers/rmt_fast_gettext'
 
 
 class SMTImporter
   attr_accessor :data_dir
   attr_accessor :no_systems
 
+  include FastGettext::Translation
+  locale = (ENV['LANG'] || ENV['LC_CTYPE']).to_s.match(/^([a-z]{2,}(_[A-Z]{2,})?)/).to_a[1] || :en
+  FastGettext.set_locale(locale.to_sym)
```

Comment:
1. How does the Rails app choose a locale? `gettext_i18n_rails` comes with `before_filter :set_gettext_locale`, but I don't think we are using this filter.
2. Can this copypasta from `rmt-cli` be moved into `initializers/rmt_fast_gettext` instead?

Reply:
will look into doing that

Reply:
Rails app and CLI scripts share `initializers/rmt_fast_gettext`, that's why I've asked how the Rails app is setting the locale (looks like it doesn't?)

Reply:
I think as we use some extra gem (Fast gettext) we have to explicitely set locale there, am I right?

Reply:
> I think as we use some extra gem (Fast gettext) we have to explicitely set locale there, am I right?

We should be using `before_filter :set_gettext_locale`, but we aren't. Leon has created a card for it... somewhere.

Reply:
ok then we merge this PR